byline-manager.php

```diff
@@ -6,9 +6,9 @@
  * Author:          Alley Interactive
  * Text Domain:     byline-manager
  * Domain Path:     /languages
- * Version:         0.2.0
+ * Version:         0.2.1
  * Requires PHP:    7.4
- * Tested up to:    4.9
+ * Tested up to:    6.2.2
```

Comment:
Since 6.3 is out now, can we update it to 6.3 instead? Did you test it on 6.3 locally?

Reply:
Not yet, I had tested it with 6.2.2.  Will test with 6.3 before merging it.